cmd/flux/create_source_git.go

```diff
@@ -119,7 +120,13 @@ For private Git repositories, the basic authentication credentials are stored in
     --url=https://github.com/stefanprodan/podinfo \
     --branch=master \
     --username=username \
-    --password=password`,
+    --password=password
+
+  # Create a source for a Git repository using azure provider
+  flux create source git podinfo \
+    --url=https://dev.azure.com/foo/bar/_git/podinfo \
+    --branch=master \
+    --provider=azure`,
```

Comment:
There is an example for different authentication types in `flux create source git` today, since `provider` is a new authentication mechanism, it felt logical to add an example for provider. Let me know if you disagree and if example with URL pattern makes better sense elsewhere.

Reply:
Thanks, leaving it as-is then.